src/components/Toast.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import * as ReactModal from "react-modal";
+import { Button } from "./Button";
+import { GoX } from "./Icons";
+
+export class Toast extends React.Component<{

```

Comment:
The `Toast` component should be generic so that we can use it in other cases. The current implementation is specific to gists. Can you please rename the `uri` property to `message` and change the `render` function to reflect this. The "Gust Created ..." message can be passed in as a property.

Reply:
Done.